tests/unit_tests/blas/level1/rot.cpp

```diff
@@ -59,8 +59,14 @@ int test(device *dev, oneapi::mkl::layout layout, int N, int incx, int incy, fp_
     using fp_ref = typename ref_type_info<fp>::type;
     const int N_ref = N, incx_ref = incx, incy_ref = incy;
 
-    ::rot(&N_ref, (fp_ref *)x_ref.data(), &incx_ref, (fp_ref *)y_ref.data(), &incy_ref,
-          (fp_scalar *)&c, (fp_scalar *)&s);
+    try {
```

Comment:
I would leave this code as is, so the exception goes through and the test fails if the library is not found.